src/cocaine/burlak/metrics/hub.py

```diff
@@ -0,0 +1,32 @@
+"""Projetct wide metrics registry."""
+
+
+class Hub(object):
+    """Metrics registry."""
+
+    def __init__(self):
+        self._system = {}
+        self._applications = {}
+
+    @property
+    def system(self):
+        return self._system
+
+    @system.setter
+    def system(self, s):
+        self._system = s
+
+    @property
+    def applications(self):
```

Comment:
Why do you need separated getters and setters? Why don't use just make `self._applications` -> `self.application`?